source/extensions/filters/http/ext_proc/ext_proc.h

```diff
@@ -114,9 +116,12 @@ class Filter : public Logger::Loggable<Logger::Id::filter>,
 
   void onGrpcClose() override;
 
+  void onMessageTimeout();
+  void sendBodyChunk(const ProcessorState& state, const Buffer::Instance& data, bool end_stream);
```

Comment:
nit: probably not worth it, but perhaps the code could be more readable if a public wrapper was introduced instead of making `sendBodyChunk()` public. Something like
```c++
void sendBufferedBody(const ProcessorState& state) {
  sendBodyChunk(state, *state.bufferedData(), true);
};
```

Reply:
Seems like a nice improvement, +1 from me unless there is a good reason to expose it all

Reply:
Ack, makes sense to me too.